docs/rest-api.md

```diff
@@ -0,0 +1,909 @@
+  </tr>
+  <tr>
+    <th rowspan="4">Response Body</th>
+    <td colspan="3">Empty if there is no more results. Otherwise:</td>
+  </tr>
+  <tr>
+    <th>Parameter</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>results</td>
+    <td>array of result set</td>
+    <td>A part of the results of the Flink job.</td>
+  </tr>
+  <tr>
+    <td>next_result_uri</td>
+    <td>string</td>
+    <td>URL for fetching the next part of results. Follow this URL if the client successfully receives the current part of the results.</td>
+  </tr>
+</table>
+
+## Get Job Status
+
+Get the status of a running job. The gateway only forwards the job status it gets from the Flink cluster without further modifications. If the session expires or the job cannot be found on the cluster, an exception will be thrown.
+
+<table>
+  <tr>
+    <th>Path</th>
+    <td colspan="3">/v1/sessions/:session_id/jobs/:job_id/status</td>
+  </tr>
+  <tr>
+    <th>Verb</th>
+    <td>GET</td>
+    <th>Response Code</td>
+    <td>200 OK</td>
+  </tr>
+  <tr>
+    <th rowspan="3">Path Parameter</th>
+    <th>Parameter</th>
+    <th colspan="2">Description</th> 
+  </tr>
+  <tr>
+    <td>session_id</td>
+    <td colspan="2">ID of the session from which the job is created.</td>
+  </tr>
+  <tr>
+    <td>job_id</td>
+    <td colspan="2">ID of the job whose status we're going to fetch.</td>
+  </tr>
+  <tr>
+    <th>Request Body</th>
+    <td colspan="3">empty</td>
+  </tr>
+  <tr>
+    <th rowspan="2">Response Body</th>
+    <th>Parameter</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>status</td>
+    <td>string</td>
+    <td>Status of the Flink job. See Flink's <a href="https://github.com/apache/flink/blob/master/flink-core/src/main/java/org/apache/flink/api/common/JobStatus.java"><code>JobStatus</code></a> for a complete list of status.</td>
+  </tr>
+</table>
+
+## Cancel a Job
+
+Cancel the running job. If the job is already canceled from the gateway before this API call does nothing. If the session expires or the job cannot be found on the cluster, an exception will be thrown.
+
+<table>
+  <tr>
+    <th>Path</th>
+    <td colspan="3">/v1/sessions/:session_id/jobs/:job_id</td>
+  </tr>
+  <tr>
+    <th>Verb</th>
+    <td>DELETE</td>
+    <th>Response Code</td>
+    <td>200 OK</td>
+  </tr>
+  <tr>
+    <th rowspan="3">Path Parameter</th>
+    <th>Parameter</th>
+    <th colspan="2">Description</th> 
+  </tr>
+  <tr>
+    <td>session_id</td>
+    <td colspan="2">ID of the session from which the job is created.</td>
+  </tr>
+  <tr>
+    <td>job_id</td>
+    <td colspan="2">ID of the job we're going to cancel.</td>
+  </tr>
+  <tr>
+    <th>Request Body</th>
+    <td colspan="3">empty</td>
+  </tr>
+  <tr>
+    <th rowspan="2">Response Body</th>
+    <th>Parameter</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>status</td>
+    <td>string</td>
+    <td>Must be <code>CANCELED</code>.</td>
+  </tr>
+</table>
```

Comment:
Is this guide generated automatically? Maybe we can copy the Flink documentation approach (that generates this automatically) or we can add something like swagger one day